.gitignore

```diff
@@ -32,6 +32,21 @@ flycheck_*
 memorydump.out
 test_file
 test_file.torrent
+/Tribler/Test/data/ec.pem
```

Comment:
Those aren't fixes, remove all gitignore modifications from this PR.

Most of these ignores shouldn't be there, the tests should be fixed to not write data to the repository instead.